premake5.lua

```diff
@@ -141,7 +144,7 @@ end
 --------------------------------------------------------------------------------
 workspace("clink")
     configurations({"debug", "release", "final"})
-    platforms({"x32", "x64"})
+    platforms({"x32", "x64", "arm64"})
```

Comment:
I don't know if mingw will have trouble with.  But since mingw isn't officially supported anyway, I'm fine with merging this and doing cleanup later if necessary.